pkg/config/config.go

```diff
@@ -622,7 +622,7 @@ func DefaultConfig() (*Config, error) {
 		},
 		ImageConfig: ImageConfig{
 			DefaultTransport: "docker://",
-			PauseImage:       "k8s.gcr.io/pause:3.2",
+			PauseImage:       "k8s.gcr.io/pause:3.4.1",
```

Comment:
We should make this a public const somewhere and re-use it in `from_1_17.go`.

Reply:
thanks for review. @saschagrunert  updated.